upfadapter/pfcp/udp/udp.go

```diff
@@ -8,64 +8,253 @@ package udp
 import (
 	"fmt"
 	"net"
+	"sync"
 	"time"
 
-	"github.com/omec-project/pfcp"
-	"github.com/omec-project/pfcp/pfcpType"
-	"github.com/omec-project/pfcp/pfcpUdp"
 	"upf-adapter/config"
 	"upf-adapter/logger"
+	"upf-adapter/types"
+
+	"github.com/wmnsk/go-pfcp/message"
+)
+
+const (
+	PFCP_PORT        = 8805
+	PFCP_MAX_UDP_LEN = 2048
```

Comment:
What bout using this value from `pfcp/udp/udp.go` instead of defining it here again?

Reply:
I'll resolve this comment so we have the conversation on the other comment that I already answered to.